### PR TITLE
Finish milestone 3

### PR DESCRIPTION
**Pull request by Greg0109**
Changelog in this commit:
- Finished project 3
- FIxed linter errors
- Add suggested changes

### CODE REVIEW

File: index.html
Comment (uryelah) [inline index.html]: - [ ] This meta tag is not complete, it should the `srink-to-fit` param as below:

`<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">`

This attribute fixes an iOS 9 bug related to the page zoom.
Please read on how to use responsive meta tags on the [Boostrap documentation here](https://getbootstrap.com/docs/4.3/getting-started/introduction/#responsive-meta-tag).

File: index.html
Comment (uryelah) [inline index.html]: - [ ] Instead of having 4 spaces for indentation please **change it to the default of 2 space indentation**, that will make your code take less space and be easier to read.

Remember the indentation for the SCSS and CSSs files too!

File: index.html
Comment (uryelah) [inline index.html]: - [ ] Consider saving your SVG images in an .image_name.svg file and then just calling it's path with the image tag in the HTML.
That will make your HTML file easier to read through and smaller, plus your repo more organized!

File: index.html
Comment (uryelah) [inline index.html]: - [ ] Let's keep your code tidy by removing repeated empty lines.
If you want to have empty lines to improve the readability of a block of code please keep it to a maximum of 1 empty line, otherwise the code will look less well formatted.

File: index.html
Comment (uryelah) [inline index.html]: - [ ] instead of using the `section` tag use the more semantic `header` tag for the header of the website.
Check [more on the header tag here.](https://www.w3schools.com/tags/tag_header.asp)

File: style.scss
Comment (uryelah) [inline style.scss]: - [ ] This style can be achieved using a Bootstrap classes. Check [the documentation](https://getbootstrap.com/docs/4.1/getting-started/introduction/), search for colors.

File: style.scss
Comment (uryelah) [inline style.scss]: - [ ] This style can be achieved using a Bootstrap classes. Check [the documentation](https://getbootstrap.com/docs/4.1/getting-started/introduction/), search for colors.

File: style.scss
Comment (uryelah) [inline style.scss]: - [ ] This style can be achieved using a Bootstrap classes. Check [the documentation](https://getbootstrap.com/docs/4.1/getting-started/introduction/), search for position.

File: style.scss
Comment (uryelah) [inline style.scss]: - [ ] This style can be achieved using a Bootstrap classes. Check [the documentation](https://getbootstrap.com/docs/4.1/getting-started/introduction/), search for sizing.

File: style.scss
Comment (uryelah) [inline style.scss]: - [ ] Let's avoid using id selectors for styling, they can create a lot of bugs as your stylesheet grows.
Try using specificity instead.

File: style.scss
Comment (uryelah) [inline style.scss]: - [ ] This style can be achieved using a Bootstrap classes. Check [the documentation](https://getbootstrap.com/docs/4.1/getting-started/introduction/), search for paddings or spacing.

File: index.html
Comment (uryelah) [reply index.html]: - [ ] Your Header nav bars should not be wrapping in any size.
Notice the search bar wrapping in a new row:
Screenshot_2021-01-25 NewsWeek


And how it should look like in wide screens:
Screenshot_2021-01-25 Newsweek(1)

- [ ] Notice also how the background of the site is gray in the original one and white in your clone. Make sure that bg color is correct.

- [  ] Make sure that the bottom navbars, the white ones, of the header take 100% of the width of the header, compare the screenshots above.

- [ ] The Weather and date and account buttons of the read area of the header should be horizontally aligned at the ends of the header and vertically aligned at the bottom of it please fix that.

----

- [ ] The 'Subscribe' button should never wrap to it's own row like it's happening on medium size screens in your clone.
Compare your version and the original one:
Screenshot_2021-01-25 NewsWeek(2)
Screenshot_2021-01-25 Newsweek(3)

---

- [ ] Fix the styling of your navbar and change the 'login' text to the correct icon in mobile sized screens.
Compare your clone behavior with the original site behavior:
Screenshot_2021-01-25 NewsWeek(4)
Screenshot_2021-01-25 Newsweek(5)

File: index.html
Comment (uryelah) [inline index.html]: - [ ] Please use the more semantic `footer` tag in here instead of `section`.

- [ ] The footer logo's size is not matching the origin one, too big on wide screens and too small on tablet and mobile, the list links are wrapping more than expected compared to their original behavior, and the text color is wrong.
Compare your footer and the original one below, and also in your browser in the responsive mode, and make sure it's closer to the original:
Screenshot_2021-01-25 Newsweek(7)
Screenshot_2021-01-25 NewsWeek(6)

**Review by uryelah — CHANGES_REQUESTED** [review]
Hi **Greg Rábago Sainz** & **Marcus Allen** , I'm Sarah your reviewer for this PR 👩‍🚀 

You guys did a great job kicking starting this milestone!

As you know the milestones 1 and 2 of this project do not require you to request for a re-review, but you still should take in the given feedback and make the necessary improvements requested in both your current code and in the code that you'll write for the next milestone.

So check my feedback carefully, make sure to apply it as requested and remember it so your code quality is even better in the next milestones!

You were really close so keep on coding and Rock on! 🎸 👨‍🎤 

# Required Changes 🏗️ 🐛 💡

-----

 **Kindly fix the pointed out items and comments below, then start working on the next milestone**

- [ ] First of all, make sure your linters are completely passing next milestone.
The linters requirements still stand in all the milestones of this project and having them passing will guarantee that your project follows the best coding standards.

- [ ] Make sure your commits follow the best committing formatting practices.
They should all be written in the imperative present mood and be capitalized:
```
// Good
Add first commit

// Bad
adding first commit
```

- [ ] Your README file contains information about a different project, The Next Web clone, instead of the current NewsWeek clone.
Please fix it so it contains only information, and links, pertaining to the current project.

- [ ] Organize your stylesheets inside a styles folder instead of having them floating at the root of the project. That will make the project more professional and the repo more organized.

- [ ] You don't need to include the Boostrap files directly in your repo to be able to use Bootstrap, you can simply use the [CDNs](https://getbootstrap.com/docs/5.0/getting-started/introduction/)(the link and script tags) in the index.html file.

Having all the files directly in your project makes it way heavier than it needs to be and the review harder too, so consider changing to the CDN in the next milestone. 

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Invalid 🛑 

- [ ] in order to review the project, your linters needs to be fixed.

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Invalid 🛑 

- [ ] in order to review the project, you need to get a review for Milestone 2

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by PhillipUg** [comment]
# Invalid Submission 🔴 

- [ ] Like the previous reviewer said, you need to get a review for milestone 2 before you move on to milestone 3. I am going to mark this pull request as invalid in your dashboard. Kindly submit milestone 2 fo review instead.

For any assistance feel free to reach out to me on slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

File: style.scss
Comment (ghost) [inline style.scss]: Avoid using custom styling where bootstrap styling is possible. Example in the above, you can use `text-dark`, `position-relative` bootstrap classes. Please check the rest of your styling as well.

**Comment by Greg0109** [comment]
> # Invalid Submission 🔴 
> 
> 
> 
> - [ ] Like the previous reviewer said, you need to get a review for milestone 2 before you move on to milestone 3. I am going to mark this pull request as invalid in your dashboard. Kindly submit milestone 2 fo review instead.
> 
> 
> 
> For any assistance feel free to reach out to me on slack as **Phillip Musiime** or
> 
>  <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

Milestone 2 was submitted right before the milestone 3 so that I would appear in your queue in that order.....

As you can see here, it is waiting for approval but it was submitted right before the 3rd oneimage

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required 🛑

### Note: I have seen you have worked on requirements for milestone 3 but please note I am reviewing for milestone 2 and you will have to make another review request for milestone 3.

## Hello team 👋,

**Great job on this milestone 👏.**

### Highlights

- **Your webpage looks like the original.**

### Improvements to note moving forward ▶️ 

#### READme

- **Add a getting started section with instructions on how to set up your project locally.**
- **The issues page link is not working.**

#### Desktop screen size

- **There is another Sponsored insights / Rankings section below the Free signup section in column three.**

#### Tablet screen size

- **The latest news and Free signup sections are not being seen.**

#### Mobile screen size

- **Fix the order in which your sections display in the mobile screen size so it matches that of the original page. For example, The Debate section should come after the Featured Stories section. You show Sponsored insights / Rankings sections after the Featured Stories.**

#### General

- **Images and image titles for all sections should be links.**
- **Check one more in the comment below.**

## Happy coding 🥂

**Comment by Greg0109** [comment]
> # Status: Changes Required 🛑
> 
> 
> 
> ### Note: I have seen you have worked on requirements for milestone 3 but please note I am reviewing for milestone 2 and you will have to make another review request for milestone 3.
> 
> 
> 
> ## Hello team 👋,
> 
> 
> 
> **Great job on this milestone 👏.**
> 
> 
> 
> ### Highlights
> 
> 
> 
> - **Your webpage looks like the original.**
> 
> 
> 
> ### Improvements to note moving forward ▶️ 
> 
> 
> 
> #### READme
> 
> 
> 
> - **Add a getting started section with instructions on how to set up your project locally.**
> 
> - **The issues page link is not working.**
> 
> 
> 
> #### Desktop screen size
> 
> 
> 
> - **There is another Sponsored insights / Rankings section below the Free signup section in column three.**
> 
> 
> 
> #### Tablet screen size
> 
> 
> 
> - **The latest news and Free signup sections are not being seen.**
> 
> 
> 
> #### Mobile screen size
> 
> 
> 
> - **Fix the order in which your sections display in the mobile screen size so it matches that of the original page. For example, The Debate section should come after the Featured Stories section. You show Sponsored insights / Rankings sections after the Featured Stories.**
> 
> 
> 
> #### General
> 
> 
> 
> - **Images and image titles for all sections should be links.**
> 
> - **Check one more in the comment below.**
> 
> 
> 
> ## Happy coding 🥂 

Thanks for the review, I'll work on the changes ASAP

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] In the Magazine section the first element has the issue date element takind more space than the assigned
image
- [ ] In the Editors pick section images look distorted.
image
- [ ] Subscribe and Newsletter elements look distorted on desktop view
image
- [ ] In the magazine the given style creates elements to distort dimensions, making the sections and elements not look as the original design, in tablet view this are the issues generated:
  - [ ] Horizontal alignemn
  - [ ] image size
  - [ ] white space around elements
  - [ ] bottom aligment
image
- [ ] In editors pick section break point starts at 768px but the clone is not responding or changing as expected
image
- [ ] In editors pick images look enlarged and titles make the grid uneven at the bottom
- [ ] in smaller screens the style of the text makes it visually cluttered, generating also that the layout breaks unevenly. 
image
- [ ] The style makes the dimensions go off, and take more space than necessary, distorting the structure of the elements.
image
- [ ] mobile view lacks white space between the margins of the page and the inside elements
image
- [ ] Subscribe and Newsletter sections have issues on small screens
image
image
- [ ] Use semantic HTML for `<main>`, `<article>`, and `<section>`
- [ ] Minimize the use of CSS rules that have a Bootstrap counter part:
  - [ ] Look for `border-radius`, `display`s, `positions`s, `margin` and `paddings`, `background` colors, `text-decoration`.
  - [ ] Keep code DRY in the stylesheet and for duplicated rules use CSS utility selectors or group all elements with the same rule. 
- [ ] There are still previous requirements from M1 and M2 that haven't been met yet, I suggest you recall and work on the issues that are left.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

<div>
  <div style="width:50px; height: 50px" align="center">
    <a href="https://avatars.githubusercontent.com/u/25774518?s=400&u=ae20ddf2d6fb3cae9e2f4044afe3b93d19e5b69e&v=4" alt="Julio Añoveros" title="@jubaan" width="100px" heigh="100px">
    </a>
  </div>
  </br>
  <p align="center">
    Julio Añoveros | @jubaan</br></br>
    <b>Full-stack Web Developer</b> and</br>
    Code Reviewer at <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a>:rocket:
  </p>
  <p align="center">
    <b>Feel stuck?</b></br>
    First breath... just do it.</br>
    Take 15-20 min away from the computer,</br>
    stretch, meditate, do some exercise, drink water,</br>
    or do whatever is necessary to distract your mind</br> 
    from the obstacle.</br></br> 
    Sometimes you just need to get away</br>
    to see things with a new perspetive.
  </p>
  <p align="center">
    <b>Need help?</b></br>
    Send me a DM in <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a> Students Slack,</br>
    you'll find me as @jubaan.
  </p>
  </br>
  <p align="center">
    <q>A <b>great programer</b> is not the one that knows it all,</br>
    but the one that ask a lot of question.</q></br></br>
    <b>Don't hesitate to ask for help!</b>
  </p>
</div>

**Comment by MarcusAl** [comment]
Edited the Magazine section, added the icon from a previous code review, and fixed spacing issues

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team. 💻 👀 ✨ 
- Good work applying many of the changes requested previously,

## However, there are some changes required 👀 🧰 🔨 🔧 
- The gray background of your Subscribe section protrudes to the left, outside of the boundaries of the main column of the site. Please fix this.
- Also there is some misalignment between the bottom of the Subscribe section and the bottom of the Newsletter section. The following screenshot shows these two problems:
image
- The top navbar is overlapped by the 'FREE SIGN-UP' and 'Sponsored insights' sections when scrolling in Tablet view. Please fix this:
image
- When resizing into mobile view, there is very little space between the Newsweek logo on the navbar and the top of the page. Also, the SUBSCRIBE button seems centered instead of aligned to the right, as it should be. This means that different mobile screen sizes will see the SUBSCRIBE button in different positions, instead of aligned to the right. Please fix these issues.
- Also, the footer shows zero paddings between the social media icons and the FREE SIGNUP button. Here's a screenshot of the two previous issues:
image
- On tablet view there is a small gap to the right of the navbar. Please fix it:
image


### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the master branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

**Pull request merged**
